scripts/zones/Jugner_Forest/mobs/Stag_Beetle.lua

```diff
@@ -1,11 +1,18 @@
 -----------------------------------
 -- Area: Jugner Forest
 --  MOB: Stag Beetle
+-- Note: PH for Panzer Percival
 -----------------------------------
+local ID = require("scripts/zones/Jugner_Forest/IDs")
 require("scripts/globals/regimes")
+require("scripts/globals/mobs")
 -----------------------------------
 
 function onMobDeath(mob, player, isKiller)
     dsp.regime.checkRegime(player, mob, 12, 1, dsp.regime.type.FIELDS)
     dsp.regime.checkRegime(player, mob, 13, 2, dsp.regime.type.FIELDS)
-end;
+end
+
+function onMobDespawn(mob)
+    phOnDespawn(mob, ID.mob.PANZER_PERCIVAL_PH, 10, 3600) -- 1 hour
```

Comment:
he has no timer, he's one of the rare probably "true lotto" or "pure lotto" - anytime ph would repop he can. even off his own death. on retail its on PH that can pop at either site, but since we don't have that feature for non NM yet multiple PH to simulate it is fine. But he's got no cooldown.

Reply:
Eh..nvm someone will inevitable argue about the "2 copies" of him being the reason. I forgot I already decided to not care...

Deleting evidence, i never posted :wink: